src/Time/TimeSteppers/RungeKutta4.hpp

```diff
@@ -0,0 +1,218 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class RungeKutta4.
+
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+#include <ostream>
+#include <pup.h>
+
+#include "ErrorHandling/Assert.hpp"
+#include "ErrorHandling/Error.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Time/Time.hpp"
+#include "Time/TimeSteppers/TimeStepper.hpp"  // IWYU pragma: keep
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+struct TimeStepId;
+namespace TimeSteppers {
+template <typename Vars, typename DerivVars>
+class History;
+}  // namespace TimeSteppers
+/// \endcond
+
+namespace TimeSteppers {
+
+/*!
+ * \ingroup TimeSteppersGroup
+ *
+ * The standard 4th-order Runge-Kutta method, given e.g. in
+ * https://en.wikipedia.org/wiki/Runge-Kutta_methods
+ * that solves equations of the form
+ *
+ * \f{eqnarray}{
+ * \frac{du}{dt} & = & RHS(t,u).
+ * \f}
+ * Given a solution \f$u(t^n)=u^n\f$, this stepper computes
+ * \f$u(t^{n+1})=u^{n+1}\f$ using the following equations:
+ *
+ * \f{eqnarray}{
+ * v^{(1)} & = & u^n + dt\cdot RHS(t^n, u^n)/2,\\
```

Comment:
[optional] In a decent amount of literature (the one pertaining to timestepping in DG/FV/FD specifically) I've seen `\mathcal{L}` used instead of `RHS`

Reply:
The LTS literature doesn't seem to have a consistent notation for this quantity, so `\mathcal{L}` seems as good as anything from my side.